maven-release-plugin/pom.xml

```diff
@@ -109,7 +109,7 @@
     <dependency>
       <groupId>org.apache.maven.plugin-testing</groupId>
       <artifactId>maven-plugin-testing-harness</artifactId>
-      <version>3.3.0</version>
+      <version>${maven-plugin-testing-harness.version}</version>
```

Comment:
This is used only once, we can safely use the literal, this is what we do everywhere.

Reply:
Ok